skyportal/handlers/api/group.py

```diff
@@ -511,10 +527,20 @@ def patch(self, group_id, *ignored_args):
         if groupuser is None:
             return self.error(f"User {user_id} is not a member of group {group_id}.")
 
-        if data.get("admin") is None:
-            return self.error("Missing required parameter: `admin`")
-        admin = data.get("admin") in [True, "true", "True", "t", "T"]
+        if data.get("admin") is None and data.get("canSave") is None:
+            return self.error(
+                "Missing required parameter: at least one of `admin` or `canSave`"
+            )
+        admin = data.get("admin", groupuser.admin) in [True, "true", "True", "t", "T"]
+        can_save = data.get("canSave", groupuser.can_save) in [
+            True,
+            "true",
+            "True",
+            "t",
+            "T",
+        ]
```

Comment:
I know it was already like this before, but perhaps it would be good to standardize what we accept as `True` for these parameters. These are more forgiving than the equivalents in the POST handler. I don't really care which one we settle on but it doesn't seem right to have the discrepancy there. Maybe just only accept what's parsed as `True` by the `data.get()` call, since that's probably the most common in the rest of the app?